core/src/Cuda/Kokkos_Cuda_Parallel_Range.hpp

```diff
@@ -463,8 +463,24 @@ class ParallelScan<FunctorType, Kokkos::RangePolicy<Traits...>, Kokkos::Cuda> {
  public:
   using pointer_type   = typename Analysis::pointer_type;
   using reference_type = typename Analysis::reference_type;
+  using value_type     = typename Analysis::value_type;
   using functor_type   = FunctorType;
-  using size_type      = Cuda::size_type;
```

Comment:
Nitpick: I would have kept that guy and not update the definitions of `m_scratch_flags` and `m_final` below.
Same comment in `ParallelScanWithTotal`.

Reply:
I went ahead and applied the suggestion to reduce the diff and be consistent with the HIP backend